### ISSUE TITLE
window innerWidth/Height, viewport sizes too huge inside p5xr.onXRFrame() on some devices

### ISSUE THREAD

**Issue by vedhant** [stalgiag/p5.xr#22]
inside of ```p5xr.onXRFrame```, I am consoling as shown below : 
```javascript
console.log("window innerWidth, innerHeight : ", window.innerWidth, window.innerHeight);
if(p5.instance.width < window.innerWidth * window.devicePixelRatio) {
  let oldWidth = p5.instance.width;
  p5.instance.resizeCanvas(
    window.innerWidth * window.devicePixelRatio,
    window.innerHeight * window.devicePixelRatio
  );
  console.log('p5 Canvas resized from '+oldWidth+' to '+p5.instance.width);
}
```
```javascript
let viewport = session.baseLayer.getViewport(view);
console.log("viewport width, height : ", viewport.width, viewport.height);
self.gl.viewport(viewport.x, viewport.y,
  viewport.width, viewport.height);
```

My original ```window.innerWidth``` = 393 and ```window.innerHeight``` = 786 but I am getting very huge values of ```window.innerWidth``` ```window.innerHeight```, ```viewport.width``` ```viewport.height``` as shown below : 

![Screenshot from 2019-05-23 00-11-18](https://user-images.githubusercontent.com/32607479/58200091-94622a00-7cef-11e9-9dfd-db61b496b94c.png)

As a result of this, my viewport is so big that the left eye portion actually covers my entire device screen (right eye is rendered outside my device screen) and the rendered images seemed magnified.

Note: On an another device I tested, this issue does not happen and both the left and right eye are rendered correctly. However, it renders correctly only if the following correction in ```p5xr.onXRFrame``` is made : 
```javascript
// if(p5.instance.width < window.innerWidth * window.devicePixelRatio) {
if(p5.instance.width < window.innerWidth) {
  let oldWidth = p5.instance.width;
  p5.instance.resizeCanvas(
    // window.innerWidth * window.devicePixelRatio,
    window.innerWidth,
    // window.innerHeight * window.devicePixelRatio
    window.innerHeight
  );
  console.log('p5 Canvas resized from '+oldWidth+' to '+p5.instance.width);
}
```

**Comment by stalgiag** [comment]
Hm it seems that is inconsistent across different devices. I added the resize canvas to fix a low resolution problem I was experiencing on my phone which has a device pixel ratio of 2.

It looks like that conditional is just circularly true on your device. I would say the pixelation is lower priority and the resizing should be removed for the time being. The pixelation can be fixed down the road.

**Comment by stalgiag** [comment]
I will remove it in the PR I am working on currently.

**Comment by stalgiag** [comment]
should be closed with #23
